### PR TITLE
3.5.3 Update

### PR DESCRIPTION
**Pull request by ViridianMelody**
[Jira issue](https://anaconda.atlassian.net/browse/PKG-1027)
[Upstream Repo](https://github.com/jupyterlab/jupyterlab)
[Changelog](https://github.com/jupyterlab/jupyterlab/releases/tag/v3.5.3)

## Changes
- Pin specific versions for `jupyter-packaging` and `packaging` in host
- Add `--no-deps` to build command

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ app:
 
 requirements:
   host:
-    - jupyter-packaging >=0.9,<2
-    - packaging
+    - jupyter-packaging 0.12.0
```
Comment (marcoesters) [inline recipe/meta.yaml:31]: This should be the lowest compatible version available, which is 0.10.4

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,8 +28,8 @@ app:
...
+    - packaging 22.0
```
Comment (marcoesters) [inline recipe/meta.yaml]: I think packaging without version is okay. I will add it to the exception list in the linter.

File: recipe/meta.yaml (hunk 1 above)
Comment (ViridianMelody) [reply recipe/meta.yaml:31]: I'm actually getting build failures with both 0.10.4 and 0.11.1, so I'm reverting back to 0.12.0.

**Review by marcoesters — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**